json-rpc/json-rpc-spec.md

```diff
@@ -170,28 +170,100 @@ if include_events is false, the events field in the Transaction object will be a
 
 
 ```
-// Request: fetches 10 transactions since version 100
-curl -X POST --data '{"jsonrpc":"2.0","method":"get_transactions","params":[100, 10, false],"id":1}'
+// Request: fetches 10 transactions since version 1000
+curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"get_transactions","params":[1000, 10, false],"id":1}'
 
 // Response
 {
-  "id":1,
-  "jsonrpc": "2.0",
-  "result": [
-    {
-      "version": 100,
-      "transaction_data": {
-         "type": "user"
-         "sender": "c94770007dda54cF92009BFF0dE90c06F603a09f",
-         "sequence_number": 10,
-         "max_gas_amount": 7000,
-         "gas_unit_price": 3,
-         "expiration_time": 1582007787665718,
-      },
-      "events": [] // empty because include_events is set to false
-    },
-    ...
-  ]
+    "id":1,
+    "jsonrpc":"2.0",
+    "result":[
+        {
+            "events":[
+
+            ],
+            "gas_used":0,
+            "transaction":{
+                "timestamp_usecs":1590615334931315,
+                "type":"blockmetadata"
+            },
+            "version":1000,
+            "vm_status":4001
+        },
+        {
```

Comment:
the result here seems to be wrong.

Also,  no need to repeat entire response.

Reply:
I guess the version I chose gave me only blockmetadataTransaction instead of UserTransaction. Will try for some higher values and see if a appropriate version gives the latter

Reply:
Updated it with 873916c